### ISSUE TITLE
Error when trying to send an email notification about a survey

### ISSUE THREAD

**Issue by markpolyak** [indico/indico#6667]
**Describe the bug**
I'm trying to send an email notification about a new survey in an event. When I press the "Send" button, nothing happens: no error messages are shown and the notification email editor window stays open. Upon examining the Network tab in developer window in the browser, I found that the server response is 422. But it is not shown anywhere on the frontend, so the user has completely no idea what is going on.

Server response to `POST https://indico.myserver-name.tld/event/1/manage/surveys/2/email/send` is 
```
{
    "webargs_errors": {
        "sender_address": [
            "Missing data for required field."
        ]
    }
}
```
This is not really informative, since the "sender_address" is chosen from a drop-down list and can't be edited. I've tried choosing different addresses from the list, the error message didn't change. 

Other email notifications such as "New submission", "New registration", etc. are working as expected on my server, so I assume that the SMTP server settings were set correctly during Indico setup. Anyway, even if the 422 error is caused by some kind of misconfiguration of my Indico instance, it would still be nice to show a meaningful error message to the user.

**To Reproduce**
1. Go to an event (a conference in my case)
2. Switch to the management area of the event
3. Go to Organization -> Surveys in the left-side menu
4. Choose a survey (or create one) and click "Manage"
5. Click the "Email" button to Contact event participants
6. Fill-in the form and click "Send"
7. See that nothing happens for the user

**Expected behavior**
Email(s) should be sent or a meaningful error message should be presented to the user.

**Screenshots**
Safari:
![2024-12-13_16-04-07](https://github.com/user-attachments/assets/8a8daf8c-3085-459e-a04a-4a26773192dd)
Chrome:
![2024-12-13_16-08-27](https://github.com/user-attachments/assets/a9ab094e-2f49-48bc-bd80-31727648c912)

**Additional context**
I've tried Safari and Chrome in case the issue is somehow browser-related. Apparently it is not, screenshots above are from different browsers.

Also I would really appreciate any help with how to get rid of the error and finally send those emails :)

**Comment by ThiefMaster** [comment]
Are you running a customized version of Indico? If yes, it sounds like you updated the server-side code but did not rebuild the assets.

You can confirm this using devtools. Check the sender address field, and see how it's called. It should be `sender_address` but my guess is that for you it's `from_address`.

![image](https://github.com/user-attachments/assets/5fc71aa2-d144-4591-ac05-bd4655966d58)

**Comment by markpolyak** [comment]
> my guess is that for you it's `from_address`.

You are right! Will try to rebuild the assets and see if the error goes away.

**Comment by Gufderald** [comment]
Ahoy! We are building a custom wheel with adding translations according to [documentation](https://docs.getindico.io/en/stable/building/), and then installing it on another instance. While building with  `build-wheel.py indico`, next lines appear in the output:
```
#20 57.23 + ./bin/maintenance/build-wheel.py indico --ignore-unclean
#20 57.33 working tree is not clean [ignored]
#20 57.33 building assets
#20 202.0 building wheel
```
So I assume, assets are built correctly.

What am I missing?

**Comment by ThiefMaster** [comment]
what output does `git status` give you? I'm curious what makes your tree unclean. just the translation?

**Comment by Gufderald** [comment]
Sure! Here it is:
```
#12 101.4 + git status
#12 101.9 Not currently on any branch.
#12 101.9 Changes not staged for commit:
#12 101.9   (use "git add <file>..." to update what will be committed)
#12 101.9   (use "git restore <file>..." to discard changes in working directory)
#12 101.9       modified:   indico/web/client/js/jquery/index.js
#12 101.9
#12 101.9 Untracked files:
#12 101.9   (use "git add <file>..." to include in what will be committed)
#12 101.9       indico/translations/ru_RU/
#12 101.9
#12 101.9 no changes added to commit (use "git add" and/or "git commit -a")
```

**Comment by ThiefMaster** [comment]
Some more commands to run and post:

```
git log --graph --pretty=short -10
grep -A2 '<FinalDropdown' indico/modules/events/persons/client/js/EmailDialog.jsx
```

**Comment by Gufderald** [comment]
```
#13 86.21 + git status
#13 86.48 Not currently on any branch.
#13 86.48 Changes not staged for commit:
#13 86.48   (use "git add <file>..." to update what will be committed)
#13 86.48   (use "git restore <file>..." to discard changes in working directory)
#13 86.48       modified:   indico/web/client/js/jquery/index.js
#13 86.48
#13 86.48 Untracked files:
#13 86.48   (use "git add <file>..." to include in what will be committed)
#13 86.48       indico/translations/ru_RU/
#13 86.48
#13 86.48 no changes added to commit (use "git add" and/or "git commit -a")
#13 86.48 + git log --graph --pretty=short -10
#13 86.48 * commit 77a495c7885b7a4a0bedd74c54e7788415495849
#13 86.48   Author: Adrian Moennich <adrian.moennich@cern.ch>
#13 86.48
#13 86.48       Release 3.3.5
#13 86.48 + grep -A2 '<FinalDropdown' indico/modules/events/persons/client/js/EmailDialog.jsx
#13 86.49         <FinalDropdown
#13 86.49           name="sender_address"
#13 86.49           label={Translate.string('Sender')}
```

**Comment by ThiefMaster** [comment]
OK, so it should all be fine. Are you 100% sure you installed the correct wheel and didn't build multiple ones and used an older one?

Check `ls -l dist/` and `pip freeze | grep indico` to see what's build and what's installed.
